install/scripts/SetupExtendedTextInteractionConfigurationRegistry.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ *
+ * @author Sergei Mikhailov <sergei.mikhailov@taotesting.com>
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\install\scripts;
+
+use oat\oatbox\extension\script\ScriptAction;
+use oat\oatbox\reporting\Report;
+use oat\taoQtiItem\model\QtiCreator\ExtendedTextInteractionConfigurationRegistry;
+
+/**
+ * Run the following to enable Math Entry format option
+ * php index.php 'oat\taoQtiItem\install\scripts\SetupExtendedTextInteractionConfigurationRegistry' -m 1
+ *
+ * Run the following to disable Math Entry format option
+ * php index.php 'oat\taoQtiItem\install\scripts\SetupExtendedTextInteractionConfigurationRegistry' -m 0
+ */
+class SetupExtendedTextInteractionConfigurationRegistry extends ScriptAction
+{
+    public const HAS_MATH = 'math';
+
+    protected function provideOptions(): array
+    {
+        return [
+            self::HAS_MATH => [
+                'prefix'      => 'm',
+                'longPrefix'  => self::HAS_MATH,
+                'description' => 'Enables / disables Math entry on extended text interactions',
+            ],
+        ];
+    }
+
+    protected function provideDescription(): string
+    {
+        return sprintf('Sets `%s` values.', ExtendedTextInteractionConfigurationRegistry::class);
+    }
+
+    protected function run(): Report
+    {
+        $setValues = [];
+
+        /** @var ExtendedTextInteractionConfigurationRegistry $registry */
+        $registry = $this->propagate(new ExtendedTextInteractionConfigurationRegistry());
```

Comment:
from my side i would have made this a .env var and use the feature flag that we already have , because this is adding 1 more config that needs deployment in order to enable this

Reply:
We don't have any real env vars on AWS right now.
`.env` vars are actually being read from the text file, which doesn't bring any benefit IMO.